src/main/java/uk/gov/companieshouse/search/api/controller/DisqualifiedSearchController.java

```diff
@@ -42,4 +46,20 @@ public ResponseEntity<Object> upsertOfficer(@PathVariable("officer_id") String o
         }
         return apiToResponseMapper.map(responseObject);
     }
+
+    @DeleteMapping("/delete/{officer_id}")
```

Comment:
Do we want this delete mapping to be more specific to disqualifications?

Reply:
It lies in the disqualified-search controller so the full path is `/disqualified-search/delete/{officer_id}`